text/0000-rfc-stages.md

```diff
@@ -0,0 +1,189 @@
+
+These edits are reviewed and merged by the relevant core team. Today, these types of changes do happen. We just don't write them down. So, merge criteria should be very loose.
+
+Major changes should have a new RFC. The old RFC is later marked "Discontinued" when the replacement is merged. 
+
+### How this applies to non-code RFCs
+
+Ember's RFC process can be used for process and work plans that are not about code. Some examples include Roadmap RFCs, this RFC itself, and changes to learning resources.
+
+When such an RFC is a Candidate for Release, the work should be shipped as described, and the result should presented to the teams with the intent of gathering feedback about whether anything is missing. If there is agreement that the work is complete, the RFC may be marked "Released" and a date is provided instead of a version.
+
+### Changes to RFC meta
+
+RFC meta is the block of text at the start of an RFC that includes data about its stage, links to relevant info, etc.
+
+Before:
+
+    - Start Date: (fill me in with today's date, YYYY-MM-DD)
+    - Relevant Team(s): (fill this in with the [team(s)](README.md#relevant-teams) to which this RFC applies)
+    - RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+    - Tracking: (leave this empty)
+
+After:
+
+    - Stage: Proposed (later updated to other stages)
+    - Release version/date: Unreleased (later updated with vX.Y.Z or YYYY-MM-DD)
+    - Start Date: (fill me in with today's date, YYYY-MM-DD)
+    - Relevant Team(s): (fill this in with the [team(s)](README.md#relevant-teams) to which this RFC applies)
+    - RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+    - Tracking: (URLs to project boards, quest issues, etc. Separate by a comma and space.)
+
+### Reconciling past RFCs
+
+For codebase-related RFCs that have already been merged, the release version is only required to be added to RFCs whose implementation was released after February 1st, 2018 (near the release of Ember 3). This is to preserve the time and effort of our contributors and volunteers.
+
+A stage will be applied to all previously merged RFCs.
+
+### Core team sign-off for Release
+
+The Core Teams' review of readiness for release has historically been an informal process. This RFC formalizes the responsibility by creating a stage for gathering confirmation that all necessary work is complete.
+
+When the work described in an RFC is ready for Release, a PR to make the change to the RFC is added by the RFC champion, and each team is requested as a reviewer. A representative of each team adds a review, and the community is also invited to comment via one week of FCP (final comment period). If a team does not respond to the request, it is assumed that the release may proceed. 
```

Comment:
How does this "one week limit" sync with the planned change to the learning team's meeting schedule to be one meeting a month? 🤔 I guess this doesn't really need to be something that **must** be discussed at a "formal" meeting, I'm only asking because we tried to keep up with an asynchronous process last time we tried to work on RFC tracking in the Learning team